### PR TITLE
SPM-1047: Add unspecified advisory type, add remapping fron newpackage to enhancement

### PR DESCRIPTION
**Pull request by semtexzv**
…cement

## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: vmaas_sync/advisory_sync.go [hunk 1]
```diff
@@ -53,6 +56,11 @@ func getAdvisoryTypes() (map[string]int, error) {
 	for _, t := range advisoryTypesArr {
 		advisoryTypes[strings.ToLower(t.Name)] = t.ID
 	}
+
+	for from, to := range advisoryTypeRemap {
+		advisoryTypes[from] = advisoryTypes[to]
+	}
+
```
Comment (Josca) [inline vmaas_sync/advisory_sync.go:63]: All these 8 new lines can be replaced by:
~~~
advisoryTypes["newpackage"] = advisoryTypes["enhancement"]
~~~

**Review by Josca — CHANGES_REQUESTED** [review]
Please mark PR title and commit with SPM-XYZ.

File: vmaas_sync/advisory_sync.go (hunk 1 above)
Comment (semtexzv) [reply vmaas_sync/advisory_sync.go:63]: I'd rather have the var on the top of the file. Having constants somewhere in  the code gets unreadable really fast.

Comment (Josca) [reply vmaas_sync/advisory_sync.go:63]: I think that the line I suggested and explaining comment would be enough, but as you wish. So please at least add SPM-XYZ to PR title to get the link in Jira.

Comment (Josca) [reply vmaas_sync/advisory_sync.go:63]: ... and to the commit message

**Review by Josca — APPROVED** [review]

**Pull request merged**